FinSearchUnified/Subscriber/Widgets.php

```diff
@@ -3,11 +3,35 @@
 namespace FinSearchUnified\Subscriber;
 
 use Enlight\Event\SubscriberInterface;
+use Enlight_Controller_Request_RequestHttp;
+use Enlight_Event_EventArgs;
 use Enlight_Hook_HookArgs;
+use Exception;
 use FinSearchUnified\Helper\StaticHelper;
+use Shopware\Components\Routing\Context;
+use Shopware\Components\Routing\Matchers\RewriteMatcher;
+use Shopware_Controllers_Widgets_Listing;
+use Zend_Cache_Core;
+use Zend_Cache_Exception;
 
 class Widgets implements SubscriberInterface
 {
+    /**
+     * @var Zend_Cache_Core
+     */
+    private $cache;
+
+    /**
+     * @var RewriteMatcher
+     */
+    private $rewrite;
+
+    public function __construct(Zend_Cache_Core $cache, RewriteMatcher $rewrite)
+    {
+        $this->cache = $cache;
+        $this->rewrite = $rewrite;
```

Comment:
Rename this to `rewriteMatcher` as `$rewrite` does not tell what it is or what it does.